index.js

```diff
@@ -14,11 +14,10 @@ const moons = [
 
   You should not need to make any other changes to the code.
 */
-function organizeMoonsByPlanet(moons) {
+const organizeMoonsByPlanet = (moons) =>{
   const result = {};
   for (let moon of moons) {
-    const name = moon.name;
-    const planet = moon.planet;
+    const {name, planet}= moon
```

Comment:
Might seem minor, but you'll improve readability if you add a space between braces and anything outside of them. (Some even put a space _inside_ the braces, like before `name` and after `planet` in the curly braces here. But not everyone does this by any means!)  Your code will be far easier to parse if you have a space before the equal sign here, and after the "fat arrow" above.

Readabiilty matters!

(But you won't lose any points here on it.)